### PR TITLE
Fix data script

### PR DESCRIPTION
**Pull request by dirkgr**
Tiny change to make the data download script work

### CODE REVIEW

File: scripts/evaluate_bidaf.sh [hunk 1]
```diff
@@ -28,24 +28,27 @@ bidaf_input=${input_file_prefix}_with_paras_${run_name}.jsonl
 if [ ! -f ${input_file_with_hits} ]; then
 	python arc_solvers/processing/add_retrieved_text.py \
 		${input_file} \
-		${input_file_with_hits}
+		${input_file_with_hits}.$$
+	mv ${input_file_with_hits}.$$ ${input_file_with_hits}
```
Comment (tusharkhot) [inline scripts/evaluate_bidaf.sh]: Aah. This is a good trick.

File: scripts/evaluate_bidaf.sh [hunk 2]
```diff
@@ -28,24 +28,27 @@ bidaf_input=${input_file_prefix}_with_paras_${run_name}.jsonl
...
 fi
 
 # Merge hits for each question
 if [ ! -f ${bidaf_input} ]; then
 	python arc_solvers/processing/convert_to_para_comprehension.py \
-	${input_file_with_hits} \
-	${input_file} \
-	${bidaf_input}
+    ${input_file_with_hits} \
```
Comment (tusharkhot) [inline scripts/evaluate_bidaf.sh]: Nit pick: Can you increase the indent ? I like having the lines in a multi-line command indented at least as much as, preferably more than, the first line of the command. Makes it clearer when a new command is starting.

File: scripts/evaluate_bidaf.sh [hunk 3]
```diff
@@ -28,24 +28,27 @@ bidaf_input=${input_file_prefix}_with_paras_${run_name}.jsonl
...
 fi
 
 # Run BiDafModel
 bidaf_output=${input_file_prefix}_qapredictions_bidaf_${model_name}_${run_name}.jsonl
 if [ ! -f ${bidaf_output} ]; then
 	python arc_solvers/run.py predict \
-				--output-file ${bidaf_output} --silent \
-				${model_dir}/model.tar.gz \
-				${bidaf_input}
+    --output-file ${bidaf_output} --silent \
```
Comment (tusharkhot) [inline scripts/evaluate_bidaf.sh]: Same here.

File: scripts/evaluate_bidaf.sh (hunk 2 above)
Comment (dirkgr) [reply scripts/evaluate_bidaf.sh]: Looks like there were some tabs vs. spaces issues going on. I fixed them.

File: scripts/evaluate_bidaf.sh (hunk 1 above)
Comment (dirkgr) [reply scripts/evaluate_bidaf.sh]: This should be done with a makefile. That would have the added benefit that when one of the earlier files changes, it knows to recompute the later ones. But I didn't want to change so much all at the same time.

**Pull request merged**